flink-connectors/flink-connector-kinesis/src/main/resources/META-INF/NOTICE

```diff
@@ -6,21 +6,29 @@ The Apache Software Foundation (http://www.apache.org/).
 
 This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt) 
 
-- com.amazonaws:aws-java-sdk-core:1.11.319
-- com.amazonaws:aws-java-sdk-kinesis:1.11.319
-- com.amazonaws:aws-java-sdk-sts:1.11.319
-- com.amazonaws:jmespath-java:1.11.319
+- com.amazonaws:amazon-kinesis-client:1.11.2
+- com.amazonaws:amazon-kinesis-producer:0.13.1
+- com.amazonaws:dynamodb-streams-kinesis-adapter:1.5.0
+- com.amazonaws:aws-java-sdk-core:1.11.603
+- com.amazonaws:aws-java-sdk-kinesis:1.11.603
+- com.amazonaws:aws-java-sdk-kms:1.11.603
+- com.amazonaws:aws-java-sdk-s3:1.11.603
+- com.amazonaws:aws-java-sdk-sts:1.11.603
+- com.amazonaws:jmespath-java:1.11.603
+- com.fasterxml.jackson.core:jackson-annotations:2.6.0
+- com.fasterxml.jackson.core:jackson-databind:2.6.7.2
+- com.fasterxml.jackson.core:jackson-core:2.6.7
+- com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.6.7
+- com.google.guava:guava:18.0
```

Comment:
this is not bundled in the jar

Reply:
ta, wasn't certain on these, so went searching to see if other NOTICE files referenced them.